[2023-09-06T12:00:38.691+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=scheduled__2023-09-05T00:00:00+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-06T12:00:38.692+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=scheduled__2023-09-05T00:00:00+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=scheduled__2023-09-05T00:00:00+00:00/task_id=create_listings_raw_table'
[2023-09-06T12:00:38.733+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=scheduled__2023-09-05T00:00:00+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-06T12:00:38.734+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=scheduled__2023-09-05T00:00:00+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=scheduled__2023-09-05T00:00:00+00:00/task_id=create_listings_raw_table'
[2023-09-06T12:00:38.766+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table scheduled__2023-09-05T00:00:00+00:00 [queued]>
[2023-09-06T12:00:38.784+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table scheduled__2023-09-05T00:00:00+00:00 [queued]>
[2023-09-06T12:00:38.786+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-09-06T12:00:38.808+0000] {taskinstance.py:1382} INFO - Executing <Task(BigQueryCreateEmptyTableOperator): create_listings_raw_table> on 2023-09-05 00:00:00+00:00
[2023-09-06T12:00:38.815+0000] {standard_task_runner.py:57} INFO - Started process 212 to run task
[2023-09-06T12:00:38.818+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Autotrader_ETL_DAG', 'create_listings_raw_table', 'scheduled__2023-09-05T00:00:00+00:00', '--job-id', '437', '--raw', '--subdir', 'DAGS_FOLDER/autotrader_etl.py', '--cfg-path', '/tmp/tmp_rh_rb7g']
[2023-09-06T12:00:38.823+0000] {standard_task_runner.py:85} INFO - Job 437: Subtask create_listings_raw_table
[2023-09-06T12:00:38.880+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=scheduled__2023-09-05T00:00:00+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-06T12:00:38.881+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=scheduled__2023-09-05T00:00:00+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=scheduled__2023-09-05T00:00:00+00:00/task_id=create_listings_raw_table'
[2023-09-06T12:00:38.884+0000] {task_command.py:415} INFO - Running <TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table scheduled__2023-09-05T00:00:00+00:00 [running]> on host 412bb32711f9
[2023-09-06T12:00:38.975+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Autotrader_ETL_DAG' AIRFLOW_CTX_TASK_ID='create_listings_raw_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-05T00:00:00+00:00'
[2023-09-06T12:00:38.990+0000] {connection.py:232} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2023-09-06T12:00:38.995+0000] {base.py:73} INFO - Using connection ID 'google_cloud' for task execution.
[2023-09-06T12:00:38.996+0000] {bigquery.py:1507} INFO - Creating table
[2023-09-06T12:00:40.233+0000] {bigquery.py:1535} INFO - Table autotrader_staging.listings_raw already exists.
[2023-09-06T12:00:40.263+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Autotrader_ETL_DAG, task_id=create_listings_raw_table, execution_date=20230905T000000, start_date=20230906T120038, end_date=20230906T120040
[2023-09-06T12:00:40.316+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-06T12:00:40.359+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
